actionRuntimes/actionProxy/README.md

```diff
@@ -1,37 +0,0 @@
-Skeleton for "docker actions"
```

Comment:
i'd like to suggest keeping this doc file and perhaps elaborating on it further as it describes how you add a new runtime container.

Reply:
Makes sense.